modules/ossm-adding-project-using-smm-resource-cli.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-create-mesh.adoc
+
+:_content-type: PROCEDURE
+[id="ossm-adding-project-using-smm-resource-cli_{context}"]
+= Adding a project to the service mesh using the ServiceMeshMember resource with the CLI
+
```

Comment:
I apologize. I missed this earlier, but our mod docs guidelines require a procedure module to have a brief introductory paragraph. Could you please add a sentence here introducing the proc?

https://redhat-documentation.github.io/modular-docs/#con-creating-procedure-modules_writing-mod-docs